Src/CSharpier.MsBuild/CSharpier.MsBuild.csproj

```diff
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <TargetFrameworks>net5.0;netstandard2.0</TargetFrameworks>
         <PackageId>CSharpier.MsBuild</PackageId>
-        <CSharpierOutputDir>../CSharpier.CLI/bin/$(Configuration)/net5.0</CSharpierOutputDir>
+        <CSharpierOutputDir>../CSharpier.Cli/bin/$(Configuration)/net5.0</CSharpierOutputDir>
```

Comment:
on an unrelated note: we should probably stop hardcoding the `net5.0`, since we're about to upgrade to net 6 for c# 10.

Reply:
Good call, I created https://github.com/belav/csharpier/issues/483 for this.